sensorprocess/sensorprocess.go

```diff
@@ -34,32 +35,16 @@ type Config struct {
 	Mutex *sync.Mutex
 }
 
-// IMUData stores the next data to be added to cartographer along with its associated timestamp so that,
-// in offline mode, data from multiple sensors can be added in order.
-type IMUData struct {
-	time time.Time
-	data cartofacade.IMUReading
-}
-
-// LidarData stores the next data to be added to cartographer along with its associated timestamp so that,
-// in offline mode, data from multiple sensors can be added in order.
-type LidarData struct {
-	time time.Time
-	data []byte
-}
```

Comment:
Instead of redefining data structs, we can use `s.TimedIMUReadingResponse` and `s.TimedLidarReadingResponse`.